pvlib/tests/iotools/test_sodapro.py

```diff
@@ -0,0 +1,157 @@
+"""
+test iotools for CAMS
+"""
+
+import pandas as pd
+import numpy as np
+import pytest
+
+from pvlib.iotools import sodapro
+from conftest import DATA_DIR, assert_frame_equal
```

Comment:
I think the CI failures are because conftest is in the parent directory, so it needs a `..` like this line in `test_psm3.py`:

https://github.com/pvlib/pvlib-python/blob/5b0b15ae2e451ddcd91af5f69f9bfb4595ec57e1/pvlib/tests/iotools/test_psm3.py#L7

Reply:
You were indeed correct! I see that Will changed this in the other functions since my last commit.